### DISCUSSION TITLE
3.4.0版本存在一些Bug，何时release新版本啊？

### DISCUSSION THREAD

**Discussion by boogaloo-jw** [HandyOrg/HandyControl#1369] [General]
3.4.0版本存在一些Bug，比如NumericUpDown控件，设置DecimalPlaces后，便无法输入，无法删除的问题，何时release新版本啊？

**Comment by NaBian** [comment]
可使用预发行版
https://www.myget.org/feed/handycontrol/package/nuget/HandyControl
